### PR TITLE
Ice Krini

### PR DESCRIPTION
**Pull request by epk**
Giving 1/3 of the icing points to @drewsuitor 

image

### CODE REVIEW

**Comment by kirinrastogi** [comment]
456556C2-C134-40B8-85C6-98DBD7190ED7-7793-0000050C03383952

File: _pages/leaderboard.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ permalink: /leaderboard/
...
 | Vikram        | 1             | 4          |
-| Kirin         | 2             | 1          |
+| Kirin         | 2             | 4          |
 | Josh          | 0             | 1          |
 | Evan          | 0             | 2          |
-| Drew          | 1             | 1          |
+| Drew          | 2             | 1          |
 | Erick         | 2             | 0          |
-| Aditya        | 0             | 1          |
+| Aditya        | 2             | 1          |
```
Comment (JackyChiu) [inline _pages/leaderboard.md]: I don't think we're doing 1L as 3 ices
```suggestion
| Aditya        | 1             | 1          |
```

Comment (vikrambombhi) [reply _pages/leaderboard.md]: I feel like a 1L should be worth more tho

Comment (samjjc) [reply _pages/leaderboard.md]: It's been worth 1 historically, if we change it for this one we'd have to change for all the past ones too

**Pull request merged**